Aztec/Classes/TextKit/TextAttachment.swift

```diff
@@ -222,9 +241,19 @@ open class TextAttachment: NSTextAttachment
             path.stroke()
         }
 
+        var imagePadding: CGFloat = 0
+        if let overlayImage = overlayImage {
+            UIColor.white.set()
+            let center = CGPoint(x: origin.x + (size.width / 2.0), y: origin.y + (size.height / 2.0))
+            let path = UIBezierPath(arcCenter: center, radius: overlayImage.size.width * 2.0/3.0, startAngle: 0, endAngle: CGFloat.pi * 2, clockwise: true)
+            path.stroke()
+            overlayImage.draw(at: CGPoint(x: center.x - (overlayImage.size.width / 2.0), y: center.y - (overlayImage.size.height / 2.0)))
```

Comment:
My 2 cents: we could really use some rounding for the *x / y* positions (otherwise the asset might look blurry!)